allure-web/src/components/execution/ExecutionView.js

```diff
@@ -29,10 +29,26 @@ class ExecutionView extends View {
         attachmentEl.parents('.step').addClass('step_expanded');
     }
 
+    calculateSteps(steps, key) {
+        return steps.map(step => {
+            var children = this.calculateSteps(step.steps, key);
+            step[key + 'Count'] = step[key].length + children.reduce((count, cur) => { return count + cur[key + 'Count'];}, 0);
+            step.steps = children;
```

Comment:
Do we actually need an assignment there?
It is almost the same value as it was before on the top line.